hexa/metrics/views.py

```diff
@@ -0,0 +1,25 @@
+import urllib.parse
+
+from django.conf import settings
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import redirect
+from django.utils import timezone
+
+from .models import HttpMethod, Request
+
+
+def save_redirect(request: HttpRequest, url: str) -> HttpResponse:
```

Comment:
I'd prefer having the redirect itself in `dashboards`, this is not purely a `metrics` feature. The `Request.objects.create` can probably be handled in the `metrics` middleware. 

Otherwise, a helper function in the `metrics` module called from `dashboards`.

This can be also done in a follow-up task if it takes too much time.

Reply:
I would argue that redirect is a service of metrics, like a url shortener: all external links should be redirected to track them?
- outbound s3 file
- link to DHIS2 page specific page